public/diory/diosphere.json

```diff
@@ -0,0 +1,81 @@
+{
+  "rooms": {
+    "/": {
+      "id": "home-room"
+    },
+    "home-room": {
+      "id": "home-room",
+      "text": "Home room",
+      "connections": [
+        {
+          "connector": "folder",
+          "address": "public/diory"
+        }
+      ],
+      "doors": [
+        {
+          "id": "image-room-id"
+        },
+        {
+          "id": "s3-room-id"
+        }
+      ]
+    },
+    "image-room-id": {
+      "id": "image-room-id",
+      "text": "Image room",
+      "connections": [
+        {
+          "connector": "folder",
+          "address": "public/diory/image-room",
+          "key": "image-room-key"
+        }
+      ],
+      "doors": [
+        {
+          "id": "first-image-room-id"
+        },
+        {
+          "id": "second-image-room-id"
+        }
+      ]
```

Comment:
I think this needs also type. Is it Local, S3, IPFS or which? Client needs that information...